csrankings.csv

```diff
@@ -20079,6 +20087,7 @@ Xiaojun Shen,University of Missouri - Kansas City,http://sce2.umkc.edu/csee/shen
 Xiaojun Wan 0001,Peking University,http://www.icst.pku.edu.cn/lcwm/wanxj,lTTeBdkAAAAJ
 Xiaojun Ye,Tsinghua University,http://www.thss.tsinghua.edu.cn/publish/soften/3131/2010/20101219111626359565665/20101219111626359565665_.html,NOSCHOLARPAGE
 Xiaojun Zhai,University of Essex,https://www.essex.ac.uk/people/zhaix77307/xiaojun-zhai,sLYyja0AAAAJ
+Xiaokang Qiu,Purdue University,https://engineering.purdue.edu/~xqiu/,xGHVK6sAAAAJ
```

Comment:
Not on the list of cross-appointed faculty in CS. https://www.cs.purdue.edu/people/faculty/index.html

Reply:
I have deleted his entry.